scipy/stats/tests/test_axis_nan_policy.py

```diff
@@ -17,6 +18,9 @@
 from scipy._lib._util import AxisError
 
 
+SCIPY_XSLOW = int(os.environ.get('SCIPY_XSLOW', '0'))
```

Comment:
Easier to skip the few tests this way than rewriting the way the cases are collected.